### PR TITLE
Userstory1

### PR DESCRIPTION
**Pull request by ArianeDahl**
Please check :)

### CODE REVIEW

File: components/FruitButton.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+border-radius: 4px;
+
+justify-content: center;
+display: flex;
+
+:hover {
+    cursor: pointer;
+    background-color: rgb(255, 243, 230);
+  }
+`;
```
Comment (Kheirah) [inline components/FruitButton.js:15]: Looks like you haven't used prettier. Please format every life before submitting your PR for a review. Formatted code really helps with readability.

File: pages/index.js [hunk 2]
```diff
@@ -1,13 +1,20 @@
+import FruitButton from "../components/FruitButton";
+import VeggieButton from "@/components/VeggieButton";
+import Header from "../components/Header";
```
Comment (Kheirah) [inline pages/index.js:3]: Be consistent with your paths. Either stick with absolute paths or relative ones. Don't mix them.

File: components/FruitButton.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+padding: 20px;
+`;
+
+
+export default function FruitButton() {
+return (
+    <CenteredContainer>
+    <ButtonComponent>
+   Fruits
+    </ButtonComponent>
```
Comment (Kheirah) [inline components/FruitButton.js:31]: The button isn't doing anything, yet. But it should be doing something. You should add an `onClick` event to render data for example. Simply having a button that doesn't do anything won't get you passed quality assurance. Better fix it now =)

**Review by Kheirah — CHANGES_REQUESTED** [review]
Good start!

Please format all your files with prettier. That'll make it easier to read.
Add functionality to your two buttons. It's not enough to have buttons; they really need to do something too.

**Review by TuncAlt — APPROVED** [review]

**Comment by TuncAlt** [comment]
Looks Good!

**Pull request merged**